examples/filesystem/README.md

```diff
@@ -1,31 +1,39 @@
 # File system usage example
 
-This is a basic example how to use a file system with RIOT in your embedded application.
+This basic example shows how to use a file system with RIOT in your embedded
+application.
 
-This example shows:
- - how to mount/format/unmount a filesystem, with spiffs, littlefs and constfs examples
- - how to open/read/write/close a file with and without newlib
+In particular, this example shows:
+
+- how to mount/format/unmount a filesystem, with spiffs, littlefs and constfs
```

Comment:
As used above, I prefer the usage of `file system`, instead of `filesystem`. Then you may reprhase: `how to mount/format/unmount a file system, either with spiffs, littlefs or constfs`.